pyonepassword/pyonepassword.py

```diff
@@ -75,9 +75,10 @@ def _run_signin(self, argv, password=None):
         return self._run(argv, OPSigninException, capture_stdout=True, input_string=password)
 
     def _run_lookup(self, argv, input_string, decode=None):
-        return self._run(argv, OPLookupException, capture_stdout=True, input_string=input_string, decode=None)
+        return self._run(argv, OPLookupException, capture_stdout=True, input_string=input_string, decode=decode)
 
-    def _run(self, argv, op_exception_class, capture_stdout=False, input_string=None, decode=None):
+    @staticmethod
+    def _run(argv, op_exception_class, capture_stdout=False, input_string=None, decode=None):
```

Comment:
Unless this is an actual bug that I'm missing, I'd like to not make this change. I understand why you'd make this static, but since I basically haven't ever used static methods, I think I'll remain consistent with that style for now. Thank you for bringing this to my attention, though. I'll give it some thought and consider using them generally.

Reply:
No, this isn't a bug. It's just not required to be a class method since 'self' is not used anywhere. That's more or less the only difference this decorator brings.
Personally, I do not see the rationale behind this, but so be it.

Reply:
I explained my rationale, and I do understand where you're coming from. I'm just not ready to make the change here.

If you want to update your other PRs per my comments, I think you have some good input. I'm prepared to take the PRs once updated.